appveyor.yml

```diff
@@ -50,4 +50,5 @@ environment:
   log_webhook_id: 382609729802862592
   SentryToken:
     secure: TSVcrUaf7lIi4YoOKrZhqrf6MvXh+XFUMO454dwB0q8ieNFOap49KhCYhw4x9sRflgQnDhppjzdbOrFiDnwQvID4rb3nXCxnJRzPLOUwOYlM4SfitMt2LnyGzyOdminq
-
+  ImgurToken:
+    secure: 4a98fd6d74e936d079ae8cad473386ca5818283c
```

Comment:
shouldn't this be `secure: PP1+bFXhjlGEbEDdKZTIgY/cwh2cZmT1HmKoSb9e4EhcM8JAbbEfQPICuY7b/tC4`?

Reply:
Also, you need to add -e "ImgurToken=$env;ImgurToken" to the docker run in the before-deploy.ps1 script

Reply:
I set up the initial token incorrectly. I created a second token with Postman (which hasn't been added yet). With the Imgur API, only the ClientID is needed for the authorization header. I think the client secret is only needed when POSTing stuff to Imgur

Reply:
That might not need to be there